### PR TITLE
Property access

### PR DESCRIPTION
**Pull request by iulyanp**
Hello,

I changed the visibility of the `$children` property and use the `getLastName` method in order for the example to work correctly.

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
👍   these fixes look good to me. Thanks!

File: components/property_access.rst [hunk 1]
```diff
@@ -245,9 +245,9 @@ can use setters, the magic ``__set()`` method or properties to set values::
     {
         public $firstName;
         private $lastName;
-        private $children = array();
+        public $children = array();
 
-        public function setLastName($name)
+        public function getLastName()
         {
             $this->lastName = $name;
```
Comment (apfelbox) [inline components/property_access.rst]: You should change this to `return $this->lastName;` then.

Comment (iulyanp) [inline components/property_access.rst]: Thanks

File: components/property_access.rst [hunk 2]
```diff
@@ -245,11 +245,11 @@ can use setters, the magic ``__set()`` method or properties to set values::
     {
         public $firstName;
         private $lastName;
-        private $children = array();
+        public $children = array();
 
-        public function setLastName($name)
+        public function getLastName()
```
Comment (xabbuh) [inline components/property_access.rst]: I don't understand this change. This section is about modifying objects not reading from them.

**Review by xabbuh — CHANGES_REQUESTED** [review]

File: components/property_access.rst (hunk 2 above)
Comment (iulyanp) [inline components/property_access.rst]: I think this looks better.

File: components/property_access.rst [hunk 3]
```diff
@@ -257,18 +257,24 @@ can use setters, the magic ``__set()`` method or properties to set values::
...
         }
 
-        // ...
+        public function getLastName() {
+            return $this->lastName;
+        }
+
+        public function getChildren() {
+            return $this->children;
+        }
```
Comment (xabbuh) [inline components/property_access.rst]: I would move both methods between `setLastName()` and `__set()`. And to match the code style of the Symfony code the opening curly brace should be on its own line. However, we can easily make this change while merging your PR.

**Review by xabbuh — APPROVED** [review]
looks good to me

**Comment by Simperfit** [comment]
This could be merged 👍 @javiereguiluz

**Review by Simperfit — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@iulyanp thanks a lot for contributing these fixes and congrats on your first Symfony Docs contribution.

**Pull request closed without merging**